setup.py

```diff
@@ -22,6 +22,7 @@
 # DEALINGS IN THE SOFTWARE.
 
 from setuptools import setup, find_packages
+import os
```

Comment:
Imports from std lib always first, then empty line and 3rd party stuff [, then empty line and own module imports].